learn/cookbooks/update_cloud.md

```diff
@@ -0,0 +1,249 @@
+# replace {updateId} with the updateId returned by the previous request
+curl \
+  -X GET 'http://<your-domain-name>/indexes/{index_uid}/updates/{updateId}' \
+  -H 'X-Meili-API-Key: API_KEY'  
+```
+
+Once the status updates to `processed`, you're good to go.
+
+## Step 3: Create the dump
+
+Before creating your dump, ensure that your [dump directory](/learn/configuration/instance_options.md#dumps-directory) is accessible. By default, dumps are created in a folder called `dumps` in the configuration file directory: `/var/opt/meilisearch/dumps`
+
+Use the following command to create a dump of your Meilisearch database:
+
+```sh
+curl \
+  -X POST 'http://<your-domain-name>/dumps' \
+  -H 'Authorization: Bearer API_KEY' 
+# -H 'X-Meili-API-Key: API_KEY' for v0.24 or below
+```
+
+```json
+{
+  "taskUid": 1,
+  "indexUid": null,
+  "status": "enqueued",
+  "type": "dumpCreation",
+  "enqueuedAt": "2022-06-21T16:10:29.217688Z"
+}
+```
+
+::: note
+The response will vary slightly depending on your version. For v0.27 and below, the response returns a dump `uid`.
+:::
+
+Use the `taskUid` to track the status of the dump with the get task endpoint:
+
+```sh
+# replace {taskUid} with the value returned in the previous response
+curl \
+  -X GET 'http://<your-domain-name>/tasks/{taskUid}' \
+  -H 'Authorization: Bearer API_KEY' 
+```
+
+Once the `status` updates to `succeeded`, move on to the next step.
+
+```json
+{
+  "uid": 1,
+  "indexUid": null,
+  "status": "succeeded",
+  "type": "dumpCreation",
+  "canceledBy": null,
+  "details": {
+    "dumpUid": "20220621-161029217"
+  },
+  "error": null,
+  "duration": "PT0.025872S",
+  "enqueuedAt": "2022-06-21T16:10:29.217688Z",
+  "startedAt": "2022-06-21T16:10:29.218297Z",
+  "finishedAt": "2022-06-21T16:10:29.244169Z"
+}
+```
+
+## Step 4: Stop the Meilisearch instance
+
+If you haven't done it yet, connect via SSH to your cloud instance and execute the following command to stop Meilisearch. Remember you may need to prefix it with `sudo` if you are not connected as root.
+
+```
+systemctl stop meilisearch
+```
+
+## Step 5: Create a backup
+
+Instead of deleting `data.ms`, we suggest creating a backup in case something goes wrong. `data.ms` can be found at `/var/lib/meilisearch/data.ms`.
+
+Move the binary of the current Meilisearch version and the database to the `tmp/` folder:
+
+```
+mv /usr/bin/meilisearch /tmp
+mv /var/lib/meilisearch/data.ms /tmp/
+```
+
+## Step 6: Install the desired version of Meilisearch
+
+Install the latest version of Meilisearch using:
+
+```sh
+# replace {meilisearch_version} with the version of your choice. Use the format: `vX.X.X`
+curl "https://github.com/meilisearch/meilisearch/releases/download/{meilisearch_version}/meilisearch-linux-amd64" --output meilisearch --location --show-error
+```
+
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+Move the new Meilisearch binary to the systemd directory containing the executable files:
```

Comment:
`/usr/bin` is the standard directory on Unix-like operating systems that contains the executable files, not the `systemd` directory.